crates/bevy_pbr/src/render/pbr.wgsl

```diff
@@ -1,60 +1,59 @@
-#import bevy_pbr::mesh_view_bindings
-#import bevy_pbr::pbr_bindings
-#import bevy_pbr::mesh_bindings
-
-#import bevy_pbr::utils
-#import bevy_pbr::clustered_forward
-#import bevy_pbr::lighting
-#import bevy_pbr::pbr_ambient
-#import bevy_pbr::shadows
-#import bevy_pbr::fog
-#import bevy_pbr::pbr_functions
+#define_import_path bevy_pbr::fragment
 
+#import bevy_pbr::pbr_functions as pbr_functions
```

Comment:
A potential followup idea: we could consider having preludes defined for Material shaders: either implicit(ex: transparently auto import `bevy_pbr::mesh_view_bindings` and `bevy_pbr::mesh_view_types`) or alternatively make them explicit (ex: allow users to call `#import bevy_render::material::prelude::*`)

Reply:
I think I like this idea of a prelude for material. That would solve the issue of people importing the pbr bindings with 2d materials

Reply:
It would probably make tools like wgsl-analyzer much harder to use though because it won't be able to know the prelude

Reply:
Coordinate with this Refactor custom import of Bevy https://github.com/wgsl-analyzer/wgsl-analyzer/pull/74
Waiting on this PR to merge before continuing this PR.

Reply:
I definitely would like a prelude/glob import system. I have a _lot_ of implicit imports from `*_bindings.wgsl` files in bevy_solari that would be a huge pain to manually specify whenever I use a binding from.

Also +1 for "auto-import prelude for Material shaders". I've seen several help messages where people are confused why they can't access a certain variable. Maybe if bevy had an editor / shader templates or something it wouldn't be a problem, but for now I think auto-importing a prelude is a good idea.